### DISCUSSION TITLE
importing from_time_and_frame_vectors()

### DISCUSSION THREAD

**Discussion by laureneastman** [skyfielders/python-skyfield#989] [Q&A] [answered]
Hi, 
I am currently trying to convert between cartesian topocentric celestial reference system  to a cartesian LTP system, and for part of that I was going to use from_time_and_frame_vectors(). However, I can't seem to be able to import this function, I've tried importing it from skyfield.vectorlib, skyfield.framelib and skyfield.positionlib and I always get an error message saying that "cannot import name 'from_time_and_frame_vectors' from XXX". How am I meant to access this function?

Many thanks, 
Lauren :)

Also, I think Skyfield is so so so so great thank you so much for making this hahaha.

**Comment by brandon-rhodes** [comment, marked as answer]
> However, I can't seem to be able to import this function

Maybe I should add an example of how to use it to the documentation. It's a method of the `ICRF` class that you can import from `skyfield.positionlib`, so you would do something like:

```
from skyfield.positionlib import ICRS
from skyfield.api import Distance, Velocity
...
d = Distance(some_vector)
v = Velocity(some_other_vector)
p = ICRS.from_time_and_frame_vectors(t, place, d, v)
```

> Also, I think Skyfield is so so so so great thank you so much for making this

Thanks for the kind words, I'm glad you're finding it useful!

**Reply by laureneastman** [reply]
That's great thank you so much! Sorry, I meant to thank you earlier but I must have gotten carried away with work!
